src/elements/content-sidebar/__tests__/MetadataSidebarRedesigned.test.tsx

```diff
@@ -1,23 +1,75 @@
 import React from 'react';
-import { IntlProvider } from 'react-intl';
+import { screen } from '@testing-library/react';
```

Comment:
all the rtl functions are available via `../../../test-utils/testing-library` so you can move screen into line 5